perception/autoware_traffic_light_selector/src/traffic_light_selector_node.cpp

```diff
@@ -63,89 +58,78 @@ void TrafficLightSelectorNode::objectsCallback(
   const sensor_msgs::msg::CameraInfo::ConstSharedPtr & camera_info_msg)
 {
   stop_watch_ptr_->toc("processing_time", true);
-  uint32_t image_width = camera_info_msg->width;
-  uint32_t image_height = camera_info_msg->height;
-  std::map<int, sensor_msgs::msg::RegionOfInterest> rough_rois_map;
+
+  const auto image_width = camera_info_msg->width;
+  const auto image_height = camera_info_msg->height;
+
+  std::map<uint8_t, RegionOfInterest> rough_rois_map;
```

Comment:
since you are going to assign first for `traffic_light_id` whose type is `int64`
```suggestion
  std::map<int64_t, RegionOfInterest> rough_rois_map;
```

Reply:
It is true. Fixed in e4418b4